### ISSUE TITLE
nvda does not announce the accentuation of the letter if the check box is Say capitalization before capitalization

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#11948]
### Steps to reproduce:
Open a text editor
In the nvda speech settings select the checkbox
Say capitalization before capitalization
return to the text editor
turn on capslook
type letters with accents
examples:
Á
À
Ã
É
Ê
Í
Ó
Ô
Ú
read the accented letters typed with the left and right arrows

### Actual behavior:
nvda informs
the character name is spoken
character accent is not spoken
is spoken capitalized

### Expected behavior:
nvda informs:
speak the name of the character
speaks the accentuation of the character
speaks  capital letter
#### NVDA installed/portable/running from source:
instaled
#### NVDA version:
nvda.exe, NVDA alpha-21524,7b92e442

#### Windows version:
windows 10 19042.662
#### Name and version of other software in use when reproducing the issue:
any text editor
#### Other information about your system:
if the checkbox
Say capitalization before capitalization
is unchecked and the user chooses another way to receive the capitalization status of the letters, everything works fine.
In this case I want the nvda to announce the capitalization of the letters, and the accentuation of the letters as well.
tested with several synthesizers and the problem remains

#### Does the issue still occur after restarting your computer?
yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
yes
#### If addons are disabled, is your problem still occuring?
yes
#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
no

**Comment by CyrilleB79** [comment]
Hello @fernando-jose-silva 

I have proposed a pull-request (#11977) that should fix this issue. Could you confirm that your issue disappear with [this test build](https://ci.appveyor.com/api/buildjobs/xnl6rsh8t20beeft/artifacts/output%2Fnvda_snapshot_pr11977-21575%2C2850e247.exe)
Thanks.

**Comment by fernando-jose-silva** [comment]
Thank you very much, the test compilation works correctly.

**Comment by ruifontes** [comment]
Helo!
This fix, I think, have broken other parts...
Go to https://www.fnac.pt/SearchResult/ResultList.aspx?SCat=0%211&Search=band&sft=1&sa=0
In the first line NVDA should read:Botão Áreas
and read:
Botão Á reas

**Comment by CyrilleB79** [comment]
Hi @ruifontes
I cannot reproduce this issue on my side with eSpeak or IBMTTS.
Please open a new issue (bug template) and fill in all the requested paragraphs. Especially the following information would be useful: detailed steps to reproduce, specific configuration, synthesizer used (try with various synth please), log in debug mode, various NVDA versions tested (beta2 / 2020.4) and the corresponding result, test without add-ons, etc.
If it appears to be a regression of 2021.1beta, please indicate it clearly and Cc NVAccess people (@feerrenrut)
Since you think the issue is link to this PR, also indicate the it with a ref to this PR.
Thanks.

**Comment by ruifontes** [comment]
Hello!
In NVDA 2020.4 the log shows:
IO - speech.speak (01:05:41.034) - MainThread (7052):
Speaking [CallbackCommand(name=say-all:lineReached), 'clicável', 'Banner Secção', 'botão', 'Áreas', CallbackCommand(name=say-all:lineReached)]
DEBUG - speech.manager.SpeechManager.speak (01:05:41.034) - MainThread (7052):
SpeechManager- speak (priority <SpeechPriority.NORMAL: 0>): [CallbackCommand(name=say-all:lineReached), 'clicável  ', 'Banner Secção  ', 'botão  ', 'Áreas  ', CallbackCommand(name=say-all:lineReached)]
DEBUG - speech.manager.SpeechManager._queueSpeechSequence (01:05:41.035) - MainThread (7052):
SpeechManager- Out Seq: [[IndexCommand(7)], ['clicável  ', 'Banner Secção  ', 'botão  ', 'Áreas  ', IndexCommand(8)], [EndUtteranceCommand()]]
DEBUG - speech.manager.SpeechManager._queueSpeechSequence (01:05:41.035) - MainThread (7052):

In NVDA 2021.1 Beta1 the log shows:
IO - speech.speech.speak (23:48:47.779) - MainThread (13444):
Speaking ['botão', 'Á', 'reas']
DEBUG - speech.manager.SpeechManager.speak (23:48:47.780) - MainThread (13444):
SpeechManager- speak (priority <SpeechPriority.NORMAL: 0>): ['botão  ', 'Á  ', 'reas  ']
DEBUG - speech.manager.SpeechManager._queueSpeechSequence (23:48:47.781) - MainThread (13444):
SpeechManager- Out Seq: [['botão  ', 'Á  ', 'reas  ', IndexCommand(24)], [EndUtteranceCommand()]]


So, the major difference is:
2020.4 -> SpeechManager- Out Seq: [[IndexCommand(7)], ['clicável  ', 'Banner Secção  ', 'botão  ', 'Áreas  ', IndexCommand(8)], [EndUtteranceCommand()]]
2021.1 Beta1 -> SpeechManager- Out Seq: [['botão  ', 'Á  ', 'reas  ', IndexCommand(24)], [EndUtteranceCommand()]]

Still necessary to create an issue?

**Comment by CyrilleB79** [comment]
Yes open a new issue with all the information required by the template and also in my previous message.
For now, I am not able to reproduce.

**Comment by CyrilleB79** [comment]
@ruifontes could you indicate if you are still experimenting the issue you were describing?

* If yes, please open a new issue as soon as possible with the detailed description. This would allow to avoid having a regression in the final release of NVDA 2021.1
* If not please also indicate it here so that to clarify this. Also if an add-on or a specific configuration was responsible for the issue, please indicate it.
Thanks a lot.
